session1/solutions/assignment3/Rakefile

```diff
@@ -1 +1,11 @@
-# assignment 3 solution
+namespace :files do
+  desc 'Rename files named *.yaml to be *.yml'
+  task :rename_yml, [:path] do |t, args|
+    path = args.path || Dir.pwd
```

Comment:
When using args parameters the correct way of assigning default value is with `args.with_defaults("blaaa")`:
http://ruby-doc.org/stdlib-2.0.0/libdoc/rake/rdoc/Rake/TaskArguments.html